lib/vdsm/supervdsm_api/udev.py

```diff
@@ -131,8 +136,8 @@ def trigger_managed_udev_rule(path):
 
 
 @expose
-def remove_managed_udev_rule(vol_id):
-    rule_file = _UDEV_RULE_FILE_NAME_MANAGED % vol_id
+def remove_managed_udev_rule(vol_id, sd_id):
+    rule_file = _UDEV_RULE_FILE_NAME_MANAGED % (sd_id, vol_id)
```

Comment:
This syntax forces any reader to scroll to the top to be able to understand what is the string in `rule_file`, and is a bit odd, relying on the old string formatting with '%' operator.

Maybe using a function that receives `sd_id` and `vol_id` as parameters, closer to where it is used and employing f-string internally would be a nice alternative.

Again, this is out of the scope of this branch.

Reply:
A separate issue will be created